pkg/spoke/controllers/manifestcontroller/manifestwork_controller_test.go

```diff
@@ -562,7 +562,7 @@ func TestAllInCondition(t *testing.T) {
 	}
 }
 
-func TestBuildManifestResourceMeta(t *testing.T) {
+func TestBuildResourceMeta(t *testing.T) {
```

Comment:
Are you going to add a test case for the new method?

Reply:
Yes, I'll add a test case for the new method.